packages/grafana-ui/src/components/Space/Space.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { css, cx } from '@emotion/css';
+import { GrafanaTheme2 } from '@grafana/data';
+import { useTheme2 } from '../../themes';
+
+export interface SpaceProps {
+  v?: number;
+  h?: number;
```

Comment:
Not _super_ enthusiastic about such a shorthand, but I think it's the easiest? Definitely open to vertical/horizontal or other suggestions :)

Reply:
can we not use the same syntax as the spacing function in the theme? an array with 1-4 elements [top, right, bottom, left]

Reply:
If you look at my example (vertical space between two block-level elements), using the spacing function syntax would make it `<Space space={[2,0,0,0]} />` which I think is a bit more cumbersome than it needs to be.

Did you have a use case in mind where the shorthand of `[2]` (4 units of spacing both horizontally vertically) or `[2, 0]` (2 units of spacing just vertically, but harder to read) would be useful?